packages/odyssey-react/src/components/Tooltip/Tooltip.module.scss

```diff
@@ -23,6 +23,10 @@
   border: 1px solid cv("gray", "900");
   opacity: 0;
   background: cv("gray", "900");
+  color: $text-body-inverse;
+  font-size: $size-body-caption;
+  font-weight: 600;
+  line-height: $title-line-height;
```

Comment:
warming my brain back up after the holiday PTO: how is it that these styles applied via the `root` class to Box are able to override the defaults? Isn't this approach brittle in the same way we've identified in the past around component load order and the CSS cascade?

Reply:
This is definitely a case of not seeing the defaults, should have:
```
color={false}
font-size={false}
font-weight={false}
line-height={false}
```
in the root Box props

Reply:
@andrewberg-okta I'm not sure I follow are you saying that we should add the code snipper above to Tooltip.tsx and its Box root? I don't see that change in the PR?

Reply:
oh nvm needed to refresh the PR 🤦